Apps/frontend/cypress/e2e/3-slider/timesweep.cy.js

```diff
@@ -0,0 +1,20 @@
+describe("timesweepSlider", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("is visible", () => {
+    Cypress._.times(10, (i) => {
+      cy.get('[data-cy="timesweepSlider-' + i + '"]').should("be.visible");
+    });
+  });
+
+  it("change value", () => {
+    cy.get(":nth-child(1) > #btn-on-off").click();
+    cy.wait(2000);
```

Comment:
Just for my understanding, why do we use the `wait` here?

Reply:
Honestly, I copied this from another test